.all-contributorsrc

```diff
@@ -142,6 +142,15 @@
       "contributions": [
         "code"
       ]
+    },
```

Comment:
please coudl you remove this. ill add you afterwards with the all contributions bot